content/en-us/reference/engine/classes/UnreliableRemoteEvent.yaml

```diff
@@ -65,31 +154,60 @@ methods:
     thread_safety: Unsafe
 events:
   - name: UnreliableRemoteEvent.OnClientEvent
-    summary: ''
-    description: ''
+    summary: |
+      Fires from a `Class.LocalScript` when either
+      `Class.UnreliableRemoteEvent:FireClient()|FireClient()` or
+      `Class.UnreliableRemoteEvent:FireAllClients()|FireAllClients()` is called on the
+      same `Class.UnreliableRemoteEvent` instance from a `Class.Script`. Not guaranteed to fire if one of the above methods are called. This can occur due to packet loss or to maintain optimal engine performance.
```

Comment:
```suggestion
      `Class.UnreliableRemoteEvent:FireClient()|FireClient()` or
      `Class.UnreliableRemoteEvent:FireAllClients()|FireAllClients()` is called on the
      same `Class.UnreliableRemoteEvent` instance from a `Class.Script`. Not
      guaranteed to fire if one of the above methods are called. This can occur due
      to packet loss or to maintain optimal engine performance.
```